src/main/java/com/flansmod/common/guns/GunType.java

```diff
@@ -500,7 +500,7 @@ protected void read(String[] split, TypeFile file) {
         super.read(split, file);
         try {
             if (split[0].equals("Damage"))
-                damage = Float.parseFloat(split[1]);
+                damage = Float.parseFloat(split[1])*FlansMod.globalWeaponDamageMultiplier;
```

Comment:
This cannot be done - on servers the multiplier is synced AFTER configs are read. IMO it needs to be applied when the damage variable is used (So in vehicle guns, ItemGun e.t.c.) And primary/secondary on vehicles that work with bullets directly, not guns.